core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -57,15 +57,15 @@ def validate_programme_risks(workbook: dict[str, pd.DataFrame]) -> list["TownsFu
                      DataFrames representing each sheet in the Round 4 submission.
     :return: ValidationErrors
     """
-    risk_programme_ids = list(workbook["RiskRegister"]["Programme ID"].dropna())
+    risk_programme_ids = workbook["RiskRegister"]["Programme ID"].dropna()
 
-    # TODO: Confirm if 1 or 3 programme risks are required and change this function accordingly
-    if len(risk_programme_ids) < 3:
+    if len(risk_programme_ids) < 1:
```

Comment:
As the options are only 1 or 0 now, using `if not risk_programme_ids` is slightly more efficient

Reply:
`risk_programme_ids` is a series so it won't evaluate to a boolean like this